src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(a) {
```

Comment:
Consider renaming the function to better reflect its behavior, for example `makeAdder` since it's not strictly infinite—it stops adding once it encounters an empty call.